### ISSUE TITLE
Emmit isn't reading suggestions correctly with screen readers

### ISSUE THREAD

**Issue by tayarndt** [microsoft/vscode#115727]
Issue Type: <b>Bug</b>

1. use a screen reader
2. type something like ! for the docktype
3. listen to the suggestions and you here the symbol you typed like !

VS Code version: Code - Insiders 1.53.0-insider (7b0cfdd04ad530a9c8d782c618afd79290be3d64, 2021-01-27T08:59:32.031Z)
OS version: Windows_NT x64 10.0.19042

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|AMD Ryzen 7 2700U with Radeon Vega Mobile Gfx   (8 x 2196)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>protected_video_decode: enabled<br>rasterization: enabled<br>skia_renderer: enabled_on<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|6.80GB (1.39GB free)|
|Process Argv|--crash-reporter-id e4c634cc-796a-446d-a837-ac02a8fa29fb|
|Screen Reader|yes|
|VM|0%|
</details><details><summary>Extensions (14)</summary>

Extension|Author (truncated)|Version
---|---|---
gitlens|eam|11.2.1
php-debug|fel|1.14.7
php-intellisense|fel|2.3.14
php-pack|fel|1.0.2
vscode-setting-export|jef|0.0.6
python|ms-|2021.1.502429796
jupyter|ms-|2020.12.414227025
remote-wsl|ms-|0.53.2
vsliveshare|ms-|1.0.3577
vsliveshare-audio|ms-|0.1.91
LiveServer|rit|5.6.1
code-settings-sync|Sha|3.4.3
vscode-ltex|val|8.4.0
markdown-all-in-one|yzh|3.4.0

</details><details>
<summary>A/B Experiments</summary>

```
vsliv695:30137379
vsins829:30139715
vsliv368cf:30146710
vsreu685:30147344
openlogontheside:30221882
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
vswsl492:30208929
wsl2prompt:30219162
vstry244:30244315
pythonvsdeb440:30224570
unusedprompt:30219164
folderexplorer:30219166
openfilemenu:30219168
pythonvsded773:30223139
pythonvspyt875:30251590
core-portspanel:30233467
coreuntitledfile:30249963

```

</details>

**Comment by tayarndt** [comment]
@isidorn

**Comment by isidorn** [comment]
@tayarndt thanks for filling this issue. I tried to reproduce this both on macOS with VoiceOver and Win with NVDA and was unable to repro. Here are my steps
1. VS Code insdiers, turn on Screen Reader
2. Open some `.html` file
3. Type `<!` notice how the `!Doctype` gets suggested and it nicely gets read out by the screen reader

Something which I also notice, if I delete the `!` and type it again `!` the suggest widget does not get auto opened and thus nothing interesting gets read out. Might you be hitting this?

@rzhao271 maybe you can also try to reproduce

**Comment by rzhao271** [comment]
Also cannot reproduce on Windows with Narrator. Typing `!` reads out "exclamation point", but that gets cut off as it shows the suggest widget, and the suggestion is read out instead.
If I hit backspace and hit `!` again then the suggest widget does not automatically re-open, like what Isidor mentioned above.
